creator-node/src/utils.js

```diff
@@ -77,9 +82,13 @@ const wait = (ms) => new Promise((resolve, reject) => setTimeout(() => reject(ne
 /** Cat single byte of file at given filepath. */
 const ipfsSingleByteCat = (path, req) => new Promise(async (resolve, reject) => {
   const start = Date.now()
-  let ipfs = req.app.get('ipfsAPI')
+  let ipfs = req.app.get('ipfsLatestAPI')
+
   try {
-    await ipfs.cat(path, { length: 1 })
+    const chunks = []
```

Comment:
why is this array of chunks necessary? it doesn't appear to be used at all, like its length isn't checked anywhere etc

Reply:
@hareeshnagaraj  strictly speaking, it's not necessary. you could get rid of this array and just put continue inside the for await ... of loop. but just to keep it simple from the ipfs docs this is the usage. but either way, you can't get rid of the for await ... of loop. that's how the cat gets called

Reply:
the docs use the array to reassemble a buffer...we aren't using it for anything. tbh it might be worth not storing unnecessary bytes in memory but also not a blocker

Reply:
@hareeshnagaraj yeah we could just call continue here and that would remove the array dependency

Reply:
yeah after our discussion, 1 byte array is totally NBD - its good either way